test/test_models.py

```diff
@@ -40,22 +52,32 @@ def get_available_video_models():
 # they are not yet supported in JIT.
 script_test_models = [
     "deeplabv3_resnet101",
```

Comment:
Why are there only three models here now?

Reply:
Because all of the classification models were taken out, because they are all being checked for scriptability.